docs/faq.rst

```diff
@@ -117,6 +117,19 @@ upstream, ``cryptography`` is also dropping support for them. To fix this issue
 you should upgrade to a newer version of OpenSSL (1.1.0 or later). This may
 require you to upgrade to a newer operating system.
 
+Installing ``cryptography`` fails with ``error: Can not find Rust compiler``
+----------------------------------------------------------------------------
+
+Building ``cryptography`` from source requires you have Rust installed and on
```

Comment:
"requires you have rust installed and available on your path" where installed and available is the link you're using elsewhere